Dockerfile

```
@@ -1,22 +1,28 @@
-FROM python:3
+FROM python:3.9 as base
```

```suggestion
FROM python:3 as base
```

Does this work so we move with python versions?

---

I'd rather be explicit about the version used

---

Well, we build every commit so my theory is that people can choose the docker container built on the python version they want if it matter.

My other theory is to just keep bandersnatch working on $latest python3 tag.